src/app/components/challenge/challengeparticipate/challengeparticipate.component.spec.ts

```diff
@@ -11,6 +11,7 @@ import { RouterTestingModule } from '@angular/router/testing';
 import { NO_ERRORS_SCHEMA } from '@angular/core';
 import { TeamlistComponent } from '../../../components/publiclists/teamlist/teamlist.component';
 import { EndpointsService } from '../../../services/endpoints.service';
+import {FormsModule} from '@angular/forms';
```

Comment:
```suggestion
import { FormsModule } from '@angular/forms';
```
(Inconsistent spacing. To avoid it in future, change your editor's `.ts` settings)

Reply:
Done